RA_Integration/RA_Integration.vcxproj

```diff
@@ -115,6 +117,10 @@
     <ResourceCompile>
       <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ResourceCompile>
+    <PostBuildEvent>
+      <Command>copy "$(OutDir)RA_Integration_d.dll"    "$(RANes_ProjectDir)"
+copy "$(OutDir)RA_Integration_d.pdb" "$(RANes_ProjectDir)"</Command>
+    </PostBuildEvent>
```

Comment:
This is the command I've been using.
```
* copy /y $(TargetPath) $(LocalDebuggerWorkingDirectory)\RA_Integration.dll
```
This will copy remove the `_d` from the dll name and put it in the directory of whatever emulator I have selected as the debugging executable.

Reply:
Sure, I put it in the RA_Integration project file because the emulator doesn't necessarily have to be rebuilt each time. I'd have to write a script depending on which emulator is being built in the future.